R/modify.R

```diff
@@ -473,3 +473,12 @@ inv_which <- function(x, sel) {
     stop("unrecognised index type", call. = FALSE)
   }
 }
+
+`list_assign2<-` <- function(x, i, value) {
```

Comment:
Interesting, I have never seen an assigner function like this before. Should it be called `list_assigner2()`, i.e. a noun? With a verb I think I'm expecting a UI like `vec_assign()`.

Reply:
oh it should probably be `list_slice2<-` actually?

Reply:
Oh yeah, duh.